wlog/logger_provider_jsonmarshal.go

```diff
@@ -41,28 +41,28 @@ func (l *jsonMapLogger) Log(params ...Param) {
 func (l *jsonMapLogger) Debug(msg string, params ...Param) {
 	switch l.level {
 	case DebugLevel:
-		l.logOutput(append(params, StringParam("message", msg), StringParam("level", "DEBUG")))
+		l.logOutput(append([]Param{StringParam("message", msg), StringParam("level", "DEBUG")}, params...))
```

Comment:
Why are params at the end now rather than the beginning? Not a big deal as long as everything works, but would like to understand if this is intentional/necessary and, if so, why it works.

Reply:
Yeah, this is intentional so that the svc1logger can overwrite the message field using a param, want its params to be applied after the ones defined here.